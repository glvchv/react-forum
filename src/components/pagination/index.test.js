import React from 'react';
import renderer from 'react-test-renderer';
import Pagination from '.';

describe('Pagination component', () => {
    it('should render component', () => {
        const tree = renderer.create(
            <Pagination postsPerPage={5} totalPosts={20} paginate={() => {}}/>
        ).toJSON();
        expect(tree).toMatchSnapshot();
    })
})