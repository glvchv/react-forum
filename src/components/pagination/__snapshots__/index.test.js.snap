// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagination component should render component 1`] = `
<ul
  className="wrapper"
>
  <li
    className="active"
    onClick={[Function]}
  >
    <a
      className="a"
      href="#!"
    >
      1
    </a>
  </li>
  <li
    className=""
    onClick={[Function]}
  >
    <a
      className="a"
      href="#!"
    >
      2
    </a>
  </li>
  <li
    className=""
    onClick={[Function]}
  >
    <a
      className="a"
      href="#!"
    >
      3
    </a>
  </li>
  <li
    className=""
    onClick={[Function]}
  >
    <a
      className="a"
      href="#!"
    >
      4
    </a>
  </li>
</ul>
`;
